<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medico_tarjeta_CitaRechazada</class>
 <widget class="QWidget" name="medico_tarjeta_CitaRechazada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>151</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F4F4F4;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_foto_solicitudes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre_solicitudes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;</string>
   </property>
   <property name="text">
    <string>Mar√≠a Fernanda cruz Zamora</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_horario_solicitudes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;
</string>
   </property>
   <property name="text">
    <string>8:30 - 9:40</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sintoma_solicitudes">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;</string>
   </property>
   <property name="text">
    <string>Dolor de Cabeza</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prioridad_solicitudes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;</string>
   </property>
   <property name="text">
    <string>Prioridad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pointPrioridad_solicitudes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F4F4;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/Ellipse7.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
